<?xml version="1.0"?>
<launch>
  <node pkg="ahl_youbot_server" type="ahl_youbot_server" name="ahl_youbot_server" output="screen">
    <param name="robot/name" value="youbot"/>
    <param name="robot/mnp_name" value="mnp"/>
    <param name="robot/yaml" value="$(find ahl_youbot_server)/yaml/youbot.yaml"/>
    <param name="robot/period" value="0.01"/>
    <param name="robot/servo_period" value="0.001"/>
    <param name="robot/use_real_robot" value="true"/>

    <param name="interface/config/ahl_cfg_yaml" value="$(find ahl_youbot_server)/yaml/ahl_youbot_config.yaml"/>
    <param name="interface/config/path" value="$(find youbot_driver)/config"/>
    <param name="interface/config/do_calibration" value="true"/>
  </node>
</launch>
